static/index.js

```diff
@@ -290,6 +304,10 @@ $(document).ready(function(){
         $("#createroom .roomname").val("");
     });
 
+    $("#clearbtn").on('click', function() {
+      $("#messages li").addClass("hidden");
+    });
```

Comment:
This should only clear the current room. I think it'd still work this way, but better to be 'more correct' and specify the room id in the selector. Resubmit request when changed